pkg/controller/operators/catalog/subscription/config.go

```diff
@@ -1,17 +1,17 @@
 package subscription
 
 import (
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/reconciler"
+	resolverCache "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/resolver/cache"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/kubestate"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorlister"
```

Comment:
Typically we keep imports groups in three sets:
- Go packages
- third party packages
- repo packages